dask/templates/dask-scheduler-ingress.yaml

```diff
@@ -24,7 +30,17 @@ spec:
     {{- end }}
     http:
       paths:
-      - backend:
-          serviceName: {{ template "dask.fullname" . }}-scheduler
-          servicePort: {{ .Values.webUI.servicePort }}
+        - path: /
+          {{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress" }}
+          pathType: {{ .Values.webUI.ingress.pathType | default "ImplementationSpecific" }}
```

Comment:
Let's not use a default value hardcoded in this template, but instead let it be part of values which needs a new entry for this.

I also think maybe Prefix is what makes most sense here?